src/date.rs

```diff
@@ -275,6 +276,20 @@ impl<Tz: TimeZone> Date<Tz> {
     pub fn naive_local(&self) -> NaiveDate {
         self.date
     }
+
+    /// Retrieve the elapsed years from now to the given [`Date`].
```

Comment:
consistency with other's function docs 
```suggestion
    /// Retrieves the elapsed years from now to the given [`Date`].
```